spec/spec_helper.rb

```diff
@@ -160,6 +162,8 @@
           t[:WEBrickSocket] ||
           # Rails connection reaper
           backtrace.find { |b| b =~ %r{lib/active_record/connection_adapters/abstract/connection_pool(/reaper)?.rb} } ||
+          # Rails connection reaper in newer Rails are native (no backtrace), but have a consistent call site
+          caller.find { |b| %r{lib/active_record/connection_adapters/abstract/connection_pool(/reaper)?.rb}.match?(b) } ||
```

Comment:
Just for the sake of consistency, let's go one or another way `b =~ %r{}` or `%r{}.match?(b)`